elements/drag-drop-elements/info.json

```diff
@@ -0,0 +1,15 @@
+{
+    "controller": "pl-drag-drop-element.py",
+    "dependencies": {
+        "nodeModulesScripts": [
+        	"jquery/dist/jquery.js",
```

Comment:
Just curious if jquery.js still has to be included here for jquery-ui.js to work, or if the default loaded version is sufficient.

Reply:
I tried doing some testing with `"jquery/dist/jquery.js",` removed, and I get some error in the console if I do that, despite all the functionality in seemingly working order. It's weird because I can confirm jquery is loaded regardless of whether we specify `"jquery/dist/jquery.js",`, even the version is the same, yet I only get errors if we remove the line.

Reply:
Roger that. It works though :) We can always come back to this if others request it.